### DISCUSSION TITLE
Fetch latest versions (launched a few hours ago)

### DISCUSSION THREAD

**Discussion by diegofss11** [storybookjs/storybook#13701] [Help]
Hey alll,

This PR contains the fix I need to support postcss 8 on storybook - https://github.com/storybookjs/storybook/releases/tag/v6.2.0-alpha.17

However, I can't fetch this version yet. When I do `npm show @storybook/react versions`, the latest version displayed is `6.2.0-alpha.14`. Any reason? Any way I can fetch this latest version or do I need to wait a bit more?

Thanks for the help,

**Comment by shilman** [comment]
Clear your npm cache? It's working for me

**Reply by diegofss11** [reply]
@shilman , thanks for the reply. Can you see the version `6.2.0-alpha.17` on your `npm show`? Cause I still can't...

**Reply by shilman** [reply]
it shows up to `6.2.0-alpha.32` for me
